backend/app/routers/elasticsearch.py

```diff
@@ -23,8 +23,8 @@ def _add_permissions_clause(
             "should": [
                 {"term": {"creator": username}},
                 {"term": {"user_ids": username}},
-                {"term": {"status": "AUTHENTICATED"}},
-                {"term": {"status": "PUBLIC"}},
+                {"term": {"status": "authenticated"}},
+                {"term": {"status": "public"}},
```

Comment:
Just curious, why capitalized doesn't work? In the database and in the router logic, it's all capitalized?

Reply:
I have fixed the problem with the files having null status. They now should always have the same status as the dataset.

I'm still not getting files in the search results for public search, looking into this.

Reply:
I don't think this has been resolved yet?

Reply:
I think this is fixed. Files should be showing up in public search now.